lib/classifier/knn.h

```diff
@@ -12,6 +12,9 @@
 
 namespace MouseTrack {
 
+/// A k-nearest neighbor classifier.
+/// For each vectory, it queries the k nearest neighbors.
+/// Each neighbor has a vote for a label, the label with the most votes wins (try to use a prime for k).
 class KnnClassifier : public Classifier {
```

Comment:
prime seems safer to me, but odd should probably be good enough, at the end, you just try to avoid ambiguity in cases where there are two or more largest voting-groups